src/java.base/share/classes/java/lang/Shutdown.java

```diff
@@ -155,12 +155,25 @@ static void halt(int status) {
     /* Invoked by Runtime.exit, which does all the security checks.
      * Also invoked by handlers for system-provided termination events,
      * which should pass a nonzero status code.
+     * <p>
+     * @implNote
+     * If the system logger {@code java.lang.Runtime} is enabled for logging level DEBUG/FINE
+     * the stack trace of the call to {@code Runtime.exit()} or {@code System.exit()}
+     * is logged.
```

Comment:
Shutdown is not a public class so this impNote won't appear in the APIs docs. Should it move to Runtime.exit and System.exit?  If it moved to a public API then "system logger" could link to System.Logger. Also would it be more correct to say "a system logger named "java.lang.Runtime" is enabled for logging levels DEBUG or FINE"?

Reply:
Roger has updated this but it's still a comment on a non-public class. I think the main question here is whether there should be a note in the System.exit and Runtime.exit to document that these methods log? If not, will it be documented anywhere, maybe a troubleshooting guide to help track down what/who is calling System.exit?

Reply:
Another way to document that could be to add the commented logger name, and  a comment, to https://github.com/openjdk/jdk/blob/master/src/java.logging/share/conf/logging.properties 
A new section at the end with some explanation on what this logger prints and how to enable it - with the appropriate wording to make it clear that it's JDK-implementation specific and not part of the spec.

Reply:
I opted to add an \@implNote in java.lang.Runtime.exit(). And create the corresponding CSR.